.pipelines/e2e-with-billing-all-regions.yml

```diff
@@ -0,0 +1,96 @@
+# Azure DevOps Pipeline running RP e2e and Billing e2e in all supported regions
+variables:
+  - template: vars.yml
+stages:
+  - stage: Sign_Off_Stage
+    displayName: Sign Off
+    dependsOn: []
+    jobs:
+    - job: Sign_off_job
+      pool:
+        name: ARO-E2E
+      steps:
+      - script: echo "Kick off E2E"
+  - template: ./templates/template-rp-and-billing-e2e-section.yml
+    parameters:
+      last_stage_of_previous_section: Sign_Off_Stage
+      e2e_subscription: $(e2e-subscription)
+      aro_v4_e2e_devops_spn: $(aro-v4-e2e-devops-spn)
+      billingE2EPipelineName: $(BillingE2EPipelineName)
+      billingE2EBranchName: $(BillingE2EBranchName)
+      gobin: ${{ variables.GOBIN }}
+      gopath: ${{ variables.GOPATH }}
+      goroot: ${{ variables.GOROOT }}
+      modulePath: ${{ variables.modulePath }}
+      locations:
+      - australiaeast
+      - southeastasia
+      - southafricanorth
+  - stage: Low_traffic_section_Summary
```

Comment:
why stagger e2es across sectors?

Reply:
We've had issues in v3.11 where we run into global quota issues if we run all regions in parallel

Reply:
what global quota issues?  We were able to run e2es in parallel across all the regions we could over the weekend.  An e2e which takes >6 hours + staggering it is not an attractive prospect.

Reply:
@jim-minter staggering also help dealing with throttling issues. When deep E2Es are enabled some of these tests will get throttled.

Reply:
Regarding:
>An e2e which takes >6 hours + staggering it is not an attractive prospect.
The goal is to run E2E per region once a day. Why does it matter if staggering happens? A group of regions can run every 4 hours.